mentoring/choosing_a_solution.md

```diff
@@ -1,79 +1,104 @@
 # Choosing a solution to mentor
 
-[video:vimeo/595885125]()
+The first decision to make as a mentor is which solution to provide student feedback. Consider your depth of knowledge and the student's abilities.
+- Are you comfortable with the exercise and the solution?
+- Can you answer the student's questions?
+- Can you meet the student's goals?
+
+Does the student's knowledge gap match your expertise?
+- If they're new to the language and looking to learn the basics you'll probably be able to help if you have any experience. 
+- If they've been programming in the language for years and trying to hit expert-level, you'll need to have a pretty solid grasp of the language yourself.
+
+The process to mentor a student is:
+1. Explore the solutions
+1. Understand the solution
+1. Ask questions about the code and solution
+1. Select the solution
+1. Select the student to mentor
+
+Video from the CEO: [Choosing a solution to mentor](https://vimeo.com/595885125)
 
 The first decision you need to make as a mentor is which solution to mentor.
 
 ## Working with the Queue
 
 You can find a list of all solutions that have been submitted for mentoring in the [Mentoring Queue](/mentoring/queue).
-The UI should look something like this:
+
+The screen looks like this:
 
 [TODO: Image of queue].
 
-If you have a particular student you'd like to mentor, you can search using the filter box, and you can order by oldest or most recent.
-On the right-hand side you can filter by language or exercise.
-You can also choose to only show exercises you've completed yourself (enabled by default), which is often wise as you might struggle to give great feedback if you've not wrestled with solving the exercise yourself.
+Once you've found a solution that looks like it might be a good fit for you to provide student feedback, take the next step and have a look at the code.  
```

Comment:
```suggestion
Once you have found a solution that looks like it might be a good fit for you to provide student feedback, take the next step and have a look at the code.  
```

Reply:
As a general rule, I try to avoid contractions with an international audience.  Unless, of course, I am teaching English in an international context.

Reply:
I generally avoid contractions as well. The contractions were in the original and I didn't want to change the tone or style.